agents_json/devopness/agents.json

```
@@ -0,0 +1,236 @@
+          "sourceId": "devopness",
+          "operationId": "addEnvironmentServer"
+        }
+      ],
+      "links": [
+        {
+          "origin": {
+            "actionId": "devopness_list_projects",
+            "fieldPath": "responses.success.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_project",
+            "fieldPath": "parameters.project_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_project",
+            "fieldPath": "responses.success.id"
+          },
+          "target": {
+            "actionId": "devopness_list_project_environments",
+            "fieldPath": "parameters.project_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_list_project_environments",
+            "fieldPath": "responses.success.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_environment",
+            "fieldPath": "parameters.environment_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_environment",
+            "fieldPath": "responses.id"
+          },
+          "target": {
+            "actionId": "devopness_list_credentials",
+            "fieldPath": "parameters.environment_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_list_credentials",
+            "fieldPath": "responses.success.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_credential",
+            "fieldPath": "parameters.credential_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_credential",
+            "fieldPath": "responses.id"
+          },
+          "target": {
+            "actionId": "devopness_create_server",
+            "fieldPath": "requestBody.credential_id"
+          }
+        }
+      ],
+      "fields": {
+        "parameters": [
+          {
+            "name": "hostname",
+            "type": "string",
+            "required": true,
+            "description": "The server name and hostname to be set on Linux cloud servers. Accepts numbers (0-9), dash (-) and lower case non accented characters. Must not be greater than 60 characters."
+          },
+          {
+            "name": "provision_input",
+            "required": true,
+            "type": "object",
+            "description": "Configuration used to launch the new server as a virtual machine instance in the cloud provider",
+            "properties": {
+              "credential_id": {
+                "type": "integer",
+                "description": "The unique id of the cloud provider credential to be used on connect to cloud provider",
+                "example": 8430015757
+              },
+              "cloud_service_code": {
+                "$ref": "#/components/schemas/ServerCloudServiceCode"
```

```suggestion
                "type": "string",
                "description": "The name of the cloud service used to provide the cloud instance",
                "example": "aws-ec2",
                "enum": [
                  "aws-ec2",
                  "digitalocean-droplet",
                  "gcp-gce",
                  "self-hosted-custom"
                ]
```

Following @yagnyaPatel's message at https://github.com/wild-card-ai/agents-json/pull/30/files#r1985705144, suggested resolution of `$ref`